sdk/samples/iot/paho_iot_hub_twin_sample.c

```diff
@@ -121,7 +121,14 @@ static void create_and_configure_mqtt_client(void)
       SAMPLE_TYPE, &env_vars, mqtt_endpoint_buffer, sizeof(mqtt_endpoint_buffer));
 
   // Initialize the hub client with the default connection options.
-  rc = az_iot_hub_client_init(&hub_client, env_vars.hub_hostname, env_vars.hub_device_id, NULL);
+  az_iot_hub_client_options options = az_iot_hub_client_options_default();
+
+  // Set the content type to JSON for Direct Method payloads and Twin Document.
+  // This option is not required to be set for JSON, since the default content type is JSON.
+  // options.method_twin_content_type =
+  // AZ_SPAN_FROM_STR(AZ_IOT_HUB_CLIENT_OPTION_METHOD_TWIN_CONTENT_TYPE_JSON);
+
+  rc = az_iot_hub_client_init(&hub_client, env_vars.hub_hostname, env_vars.hub_device_id, &options);
```

Comment:
Good point.  I'd like the user to be aware of it, so those needing the option for CBOR would be the only ones who might care to also use an option for JSON, since normal users of JSON wont care about this option.